src/main/webapp/view/transcript/transcript.js

```diff
@@ -171,3 +173,43 @@ function scrollToTopOfTranscript(transcriptLine) {
   const ulElementOffset = transcriptLine.parentElement.offsetTop;
   transcriptContainer.scrollTop = transcriptLine.offsetTop - ulElementOffset;
 }
+
+/**
+ * Returns the next transcript line based on `currentTimeMs`.
+ */
+function getNextTranscriptLine(currentTimeMs) {
```

Comment:
nit: wouldn't a better name be getTranscriptLineWithTime or transcriptLineWithTime. I don't understand how you are getting the next transcript line.

Reply:
updated to be `transcriptLineWithTime`, thanks!